src/types.ts

```diff
@@ -50,116 +67,33 @@ export type QuestionType = (typeof questionTypes)[number];
 
 export type DatePickerType = 'both' | 'calendar' | 'timer';
 
-export interface Schema {
-  name: string;
-  pages: Array<{
-    label: string;
-    sections: Array<{
-      label: string;
-      isExpanded: string;
-      reference?: FormReference;
-      questions: Array<{
-        id: string;
-        label?: string;
-        value?: string;
-        type: string;
-        required?: string | boolean | RequiredFieldProps;
-        questionInfo?: string;
-        questionOptions: {
-          type?: string;
-          concept?: string;
-          answers?: Array<Record<string, string>>;
-          rendering: RenderType;
-          max?: string;
-          min?: string;
-          conceptMappings?: Array<Record<string, string>>;
-        };
-        questions?: Array<Question>;
-        validators?: Array<Record<string, string>>;
-      }>;
-    }>;
-  }>;
-  processor: string;
-  uuid: string;
-  encounterType: string;
-  referencedForms: Array<ReferencedForm>;
-  version?: string;
-  description?: string;
-  encounter?: string | OpenmrsEncounter;
-  allowUnspecifiedAll?: boolean;
-  defaultPage?: string;
-  readonly?: string | boolean;
-  inlineRendering?: 'single-line' | 'multiline' | 'automatic';
-  markdown?: unknown;
-  postSubmissionActions?: Array<{ actionId: string; config?: Record<string, unknown> }>;
-  formOptions?: {
-    usePreviousValueDisabled: boolean;
-  };
-  translations?: Record<string, string>;
-}
+// Using extended FormBuilderSchema instead of FormSchema
+export type Schema = FormBuilderSchema;
```

Comment:
We should just directly import all types from the form-engine-lib itself and remove them from this file, we shouldn't need to export from here. Even though that would mean that we need to update a lot of files, it just seems redundant to export them from here again